cargo-near/src/commands/new/new-project-template/.github/workflows/deploy-staging.yml

```diff
@@ -34,8 +34,12 @@ jobs:
             send
 
       - name: Install cargo-near CLI
-        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/cargo-near/releases/download/cargo-near-v0.6.2/cargo-near-installer.sh | sh
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/cargo-near/releases/download/cargo-near-vcargo-near-new-ci-tool-version-self/cargo-near-installer.sh | sh
       - name: Deploy to staging
+        env:
+          # WASM reproducibility check akin to SourceScan won't be available for staging contracts, deployed from PRs, 
+          # as pull request git refs `refs/pull/NUMBER/merge` are somewhat harder to access and live only as long as PRs do
+          CARGO_NEAR_REPRODUCIBLE_DISABLE_PUSHED_TO_REMOTE_GIT_CHECK: true
```

Comment:
I could not remember why I did not review this PR, but then I remembered that I did and raised the concern offline, so here I am again.

This env variable looks like a hack to me. I'd rather consider having a flag `--allow-dirty` that will use the local folder instead of cloning it from remote.

What do you think?

Reply:
`--skip-git-remote-check` then?, because the repo is clean at the moment of build and dirty isn't the problem

Reply:
I mean, I would love to have a command that would build from my project folder (not cloning it, not checking remote), and I think with this command we can kill two birds with one stone

Reply:
that would be longer to implement and more complex code overall, than just `--skip-git-remote-check`.
primary case is still when source code is available for docker build. 
`--skip-git-remote-check` is specifically for this bump with temporary branches in prs for staging builds in workflows, which shouldn't be a frequent case